pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -124,6 +125,8 @@ type OperationExecutor interface {
 func NewOperationExecutor(
 	operationGenerator OperationGenerator) OperationExecutor {
 
+	util.RegisterMetrics()
```

Comment:
since registering same metrics multiple times is fatal error (will cause program to crash).  How about we move the registration part to `Init` hook of `util/metrics` itself?